sdcm/nemesis.py

```diff
@@ -466,6 +467,27 @@ def disrupt_rolling_restart_cluster(self):
             raise UnsupportedNemesis('RollingRestartCluster is supported only for kubernetes backends')
         self.cluster.rollout_restart()
 
+    def disrupt_authenticator_switch_and_restart(self):
+        self._set_current_disruption('AuthenticatorSwitchAndRestart %s' % self.target_node)
+        if not self._is_it_on_kubernetes():
+            raise UnsupportedNemesis('AuthenticatorSwitchAndRestart is supported only for kubernetes backends')
+        # Switch to com.scylladb.auth.SaslauthdAuthenticator
+        if self.cluster.params.get('prepare_saslauthd'):
+            if 'PasswordAuthenticator' in self.cluster.params.get('authenticator'):
+                update_authenticator(self.cluster.nodes, 'com.scylladb.auth.SaslauthdAuthenticator')
+
+        # Run connect a new session after authenticator switch, and run a short workload
+        self._prepare_test_table(ks='keyspace_for_authenticator_switch', table='standard1')
```

Comment:
Where is the "short workload" that is written in the comment?

Reply:
I mean the workload inside _prepare_test_table():

- cassandra-stress write n=400000 cl=QUORUM -port jmx=6868 -mode native cql3

Reply:
ok.